frontend/tests/e2e/search/features/work-flow-service-NegativeScenarios.feature

```diff
@@ -0,0 +1,149 @@
+@6146
+
+Feature: Application Workflow - Negative Path Scenarios
```

Comment:
I'm curious how we intend to test these scenarios. We could test them locally by manipulating the API or the frontend code but in a deployed environment I'm not sure how we would do that, or if we could do it in an automated way